### DISCUSSION TITLE
bot not responding to commands

### DISCUSSION THREAD

**Discussion by Kohlla** [Rapptz/discord.py#9321] [Q&A] [answered]
created the simplest discord bot
there was a problem, the bot only responds to commands in private messages, on the server it does not react at all
changed rights issued roles, 0 reactions

used this code

**import discord
from discord.ext import commands

intents = discord.Intents.default()
intents.message_content = True
intents = discord.Intents.default()
client = commands.Bot(command_prefix='.', intents=intents)

@client.event
async def on_ready():
    await client.change_presence(status=discord.Status.online, activity=discord.Game('testing'))
    print('Bot is ready.')

@client.command()
async def hi(ctx):
    await ctx.send('Hello!')

client.run('token')**

**Comment by Midnight145** [comment]
Don't know for sure, but you reset intents to default right after enabling message content, which doesn't look correct.

**Reply by Kohlla** [reply]
> Don't know for sure, but you reset intents to default right after enabling message content, which doesn't look correct.

can you tell me how it should be?

**Reply by j2nx** [reply]
![11525bfafe71d1fca74ad77bbd355acd.png](https://user-images.githubusercontent.com/119700884/229174559-e2210104-6549-4d09-9a1d-b84bb71555d3.png)

**Comment by Midnight145** [comment, marked as answer]
```python
import discord
from discord.ext import commands

intents = discord.Intents.default()
intents.message_content = True

client = commands.Bot(command_prefix='.', intents=intents)

@client.event
async def on_ready():
await client.change_presence(status=discord.Status.online, activity=discord.Game('testing'))
print('Bot is ready.')

@client.command()
async def hi(ctx):
await ctx.send('Hello!')

client.run('token')
```

**Reply by Kohlla** [reply]
thanks a lot it solved my problem
you are cool dude

**Comment by j2nx** [comment]
boyyyy😭😭😭😭

**Reply by Kohlla** [reply]
![wuop (12)](https://user-images.githubusercontent.com/129226241/229220198-8ce9325f-c619-471b-9fa3-c3e3cdae76e6.png)
